projects/web-wallet/rpc-protocol.md

```diff
@@ -0,0 +1,330 @@
+# 1. Introduction
+
+This document outlines the remote procedure call (RPC) methods for a decentralized application (dApp) to interact with Hathor wallets.
+
+The design is currently a minimum viable product (MVP), addressing only known needs from our community. It should be noted that this API design may change as the platform evolves and additional needs arise.
+
+The idea here is to partially replicate the headless wallet API as it is already being used by many partners and has been validated to work on their use cases
+
+# 2. Reference-level explanation
+
+## 2.1 RPC Methods
```

Comment:
I missed methods to mint/melt tokens. Such methods might be useful for dApps administrative tasks, for example.

Reply:
For the initial version, they don't seem very important. I do not see a use case of creating/minting/melting tokens through the web wallet. I think we should have them in the future, but I'm ok having v1 without them.

Reply:
Ok.